3 - FightClub/Code/FightClub.cs

```diff
@@ -7,17 +8,88 @@ public static class FightClubGame
 	{
 		public static IPowerUp PurchasePowerUp()
 		{
-			throw new NotImplementedException();
-		}
+            var powerUp = new PowerUp
+            {
+                DamageBuff = 1
+            };
+            return powerUp;
```

Comment:
If you wanted to, you could also just directly return the new object like `return new PowerUp { .. };`